src/components/Map/MainMap.js

```diff
@@ -212,7 +212,23 @@ class MainMap extends Component {
                             </Fragment>
                             }
                         </Col>
-                    </Row>
+                    </Row>}
+                    {/* Loading finished but indicatorsForCounty is undefined */}
+                    {!isLoading && indicatorsLen === 0 && 
```

Comment:
Do `isLoading` and `dataLoaded` serve the same purpose?

Reply:
`isLoading` used to indicate if it was still loading (slowly) but basically the same thing. For consistency, I'm going to make it match the other files.